db/migrate/20171214174355_remove_entity_indexes_from_user_requets.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveEntityIndexesFromUserRequets < ActiveRecord::Migration
```

Comment:
It is possible -- although does perhaps involve more `rake db:rollback ` kung-fu than most people like to do -- to remove the index from the original migration so you don't end up with a bunch of extra migrations...not that it really matters in the end, but i'm happy to explain how to do that....it's kind of like the database equivalent of a git rebase :)